android/src/main/res/anim/rns_identity.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I suggest renaming this file to a more human name. I'm aware that `identity` is the true 'correct' name for this but I'm keen on calling it something less smart but more approachable e.g. `no_animation`, `do_nothing`